battdat/postprocess/tagging.py

```diff
@@ -127,9 +127,16 @@ class AddState(RawDataEnhancer):
     """
     column_names = ['current']
 
-    def enhance(self, data: pd.DataFrame) -> None:
+    def enhance(self, data: pd.DataFrame, rest_curr_threshold: float = 1.0e-04) -> None:
```

Comment:
Could you make the variable a class attribute set via the initializer rather than a argument?

That way...
1. We match the interface for the base class
2. The value can be stored if we save the class
3. It helps ensure we use the save value every time we call the enhancer

Reply:
Kind of like how we do it here: https://github.com/ROVI-org/battery-data-toolkit/blob/main/battdat/postprocess/integral.py#L54